touchforms/formplayer/templates/formplayer/fullform-ui/templates.html

```diff
@@ -47,59 +47,121 @@ <h1 class="title" data-bind="text: title"></h1>
-                                data-bind="value: xpath" />
-                            <span class="input-group-btn">
-                                <button class="btn btn-default" id="evaluate-button">Evaluate</button>
-                            </span>
+                    <div role="tabpanel" class="tab-pane" id="debugger-evaluate">
+                        <div data-bind="template: {
+                            name: 'debugger-eval-ko-template',
+                            data: evalXPath }">
                         </div>
                     </div>
-                </form>
-            </section>
+                    <div role="tabpanel" class="tab-pane" id="debugger-settings">
+                        <form class="form-horizontal debug-controls">
+                            <div class="form-group">
+                                <div class="controls" data-bind="css: Formplayer.Const.CONTROL_WIDTH">
+                                    <label class="checkbox">
+                                        <input id="auto-sync-control" type="checkbox" checked="checked">{% trans "Auto Sync" %}
+                                        <span class="help-block">
+                                        {% blocktrans %}
+                                            When this is checked, the CloudCare Debug tool will auto sync your XML after every question answered. On slow connetions, uncheck this.
+                                        {% endblocktrans %}
+                                        </span>
+                                    </label>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
         </div>
     {% endif %}
 </script>
 
+<script type="text/html" id="debugger-eval-ko-template">
+    <form class="form-horizontal" data-bind="submit: evaluate">
+        <div class="form-group">
+            <div class="controls col-sm-12">
+                <textarea
+                    id="evaluate-result"
+                    type="text"
+                    style="display: block;"
+                    readonly
+                    data-bind="
+                        value: result,
+                        css: { 'text-error': !success(), 'form-control': true }
+                "></textarea>
+            </div>
+        </div>
+        <div data-bind="css: Formplayer.Const.LABEL_OFFSET + ' ' + Formplayer.Const.CONTROL_WIDTH" >
+            <div class="input-group">
+                <input
+                    id="xpath"
+                    class="form-control"
+                    name="xpath"
+                    placeholder="/data/[your path here]"
+                    data-bind="value: xpath" />
+                <span class="input-group-btn">
+                    <button class="btn btn-default" id="evaluate-button">Evaluate</button>
```

Comment:
missing a trans here (i don't think it matters since this is mostly an internal tool for now, but just noting since the rest of the file seems to be translated)